config/peerpods/aws-VM-image-create-job.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: aws-image-creation
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  parallelism: 1
+  completions: 1
+  backoffLimit: 1
+  template:
+    metadata:
+      name: aws-image-creation
+    spec:
+      volumes:
+      - name: shared-data
+        emptyDir: {}
+        volumes:
+      - name: image-id
+        emptyDir: {}
+
+      securityContext:
+        runAsUser: 0 # needed for container mode dnf access
+
```

Comment:
It seems that only the "create" containers actually needs that. The other ones ("payload" and "result") can run unprivileged, right ? If so, please move the security context to the "create" container.

Reply:
Not sure about the meaning of your :+1: since `securityContext`  is still set at the POD level

Reply:
oops, correct, i got you wrong, I'll fix